### ISSUE TITLE
Download Complete notification Bugged

### ISSUE THREAD

**Issue by maulik9898** [tonyofrancis/Fetch#400]
When the download is completed and there are other downloads running then download complete notification get bugged. Notification is reappearing after dismissed.

here is video to understand clearly

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/21076623/60084034-68a1ec00-9754-11e9-9422-643e564917d8.gif)

**Comment by iramgarhia** [comment]
bro can u provide me source code beacuse m facing error

**Comment by maulik9898** [comment]
@iramgarhia what type of eeror you are facing ..? Please elaborate

**Comment by tonyofrancis** [comment]
@iramgarhia please provide sample code or a better example.

**Comment by tonyofrancis** [comment]
Are you using the default notification manager or a custom notification manager?

**Comment by maulik9898** [comment]
the default one

**Comment by maulik9898** [comment]
The default one
here is my configuration 
```

final OkHttpClient client = new OkHttpClient.Builder().build();
        final Downloader customDownloader = new CustomDownloader(client);
        final FetchConfiguration fetchConfiguration = new FetchConfiguration.Builder(this)
                .enableRetryOnNetworkGain(true)
                .setDownloadConcurrentLimit(Integer.parseInt(sharedPreferences.getString("simultaneous_download","9")))
                .createDownloadFileOnEnqueue(true)
                .preAllocateFileOnCreation(true)
                .setNotificationManager(new DefaultFetchNotificationManager(this))
                .setProgressReportingInterval(1000)
                .setHttpDownloader(customDownloader)
                .build();

        fetch = Fetch.Impl.getInstance(fetchConfiguration);
        fetch.addListener(fetchListener);

```

**Comment by tonyofrancis** [comment]
@maulik9898 Working on a better notification manager. I will post back here soon.

**Comment by tonyofrancis** [comment]
@maulik9898  This issue is now fixed in new Fetch version 3.0.10/Androidx 3.1.4 . Note: Updates have been make to the default notification manger class. Please see changelog and sample app DownloadsListAcivity for an example.

**Comment by maulik9898** [comment]
@tonyofrancis  now working correctly
